testing/liberasurecode/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Contributor: Ehud Kaldor <ehud@unfairfunction.org>
+# Maintainer: Ehud Kaldor <ehud@unfairfunction.org>
+pkgname=liberasurecode
+pkgver=1.5.0
+pkgrel=0
+pkgdesc="Erasure Code API library written in C with pluggable Erasure Code backends"
+url="https://github.com/openstack/liberasurecode"
+arch="all"
+license="COPYING" 
+depends=""
+makedepends="libtool make automake autoconf gcc musl-dev zlib-dev"
```

Comment:
you dont need `make`, `gcc` or `musl-dev` in make depends as those are pulled in by the implicit `build-base`. Please remove those.